incidenter/height.py

```diff
@@ -0,0 +1,41 @@
+from os import getenv
```

Comment:
Prefer importing modules, not classes and functions, unless there's a readability advantage
```suggestion
import os
```

Reply:
i actually like how it is now - when you see a `getenv`, adding the `os.` adds nothing.